cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/AppFabricServiceManager.java

```diff
@@ -160,6 +167,14 @@ public void updateServiceLogLevels(Map<String, LogEntry.Level> logLevels) {
     }
   }
 
+  /**
+   * Reset the log levels of app fabric service.
+   * The log levels will be reset back to when the service was launched.
+   * Note that resetting the log levels of app fabric service will also reset log levels for cdap master
+   * and dataset.service.
+   *
+   * @param loggerNames The set of logger names to be reset, if empty, all log levels will be reset.
```

Comment:
all log levels ? or all loggers ?  should it be `all loggers will be reset to the level when the service started`